CMakeLists.txt

```diff
@@ -37,9 +38,13 @@ set(USE_PYTHON_MAJOR ${PYTHON_VERSION_MAJOR}
     CACHE STRING "Set to 2 or 3 to pick your favorite Python.")
 message("-- But we will configure boost for python${USE_PYTHON_MAJOR}")
 
-find_package(Boost COMPONENTS
-    python${USE_PYTHON_MAJOR} numpy${USE_PYTHON_MAJOR}
-    serialization iostreams filesystem REQUIRED)
+# find_package(Boost COMPONENTS
+#     python${USE_PYTHON_MAJOR} numpy${USE_PYTHON_MAJOR}
+#     serialization iostreams filesystem REQUIRED)
+
+ find_package(Boost COMPONENTS
+     python numpy
+     serialization iostreams filesystem REQUIRED)
```

Comment:
Can you instead have USE_PYTHON_MAJOR default to "", so it can still be used to override?